features/admin.userstores.v1/pages/user-stores.tsx

```diff
@@ -36,9 +37,11 @@ import React, { FunctionComponent, ReactElement, useEffect, useState } from "rea
 import { useTranslation } from "react-i18next";
 import { useDispatch, useSelector } from "react-redux";
 import { Dispatch } from "redux";
-import { DropdownProps, Icon, PaginationProps } from "semantic-ui-react";
+import { Dropdown, DropdownItemProps, DropdownProps, Icon, PaginationProps } from "semantic-ui-react";
+import { UserAccountTypes } from "../../admin.users.v1/constants";
```

Comment:
Use package scoped imports for features.
```suggestion
import { UserAccountTypes } from "@wso2is/admin.users.v1/constants";
```

Reply:
This is addressed